vpr/src/analytical_place/ap_netlist.cpp

```diff
@@ -58,8 +58,11 @@ APBlockId APNetlist::create_block(const std::string& name, const t_pack_molecule
 void APNetlist::set_block_loc(const APBlockId id, const APFixedBlockLoc& loc) {
     VTR_ASSERT_SAFE(valid_block_id(id));
 
-    // Check that the location is fixed, if all values are -1 then it is not fixed.
-    if (loc.x == -1 && loc.y == -1 && loc.sub_tile == -1 && loc.layer_num == -1)
+    // Check that the location is fixed, if all dims are unfixed then it is not fixed.
```

Comment:
nit:  , should be a ;
(sorry for the nit, can't help myself!)

Reply:
...omg you're right. I can't believe I messed that one up; they are two independent clauses! Good catch!